File: CSharp/SpaceVIL/SpaceVIL/Items/Frame.cs
﻿using System.Drawing;

namespace SpaceVIL
{
    public class Frame : VisualItem
    {
        static int count = 0;
        public Frame()
        {
            SetItemName("Frame_" + count);
            SetSizePolicy(SizePolicy.Expand, SizePolicy.Expand);
            count++;
        }

        protected internal override bool GetHoverVerification(float xpos, float ypos)
        {
            return false;
        }
    }
}

File: Java/SpaceVIL/spacevil/src/main/java/com/spvessel/Frame.java
package com.spvessel;

import com.spvessel.Common.DefaultsService;

public class Frame extends Prototype {
    private static int count = 0;

    public Frame() {
        setItemName("Frame_" + count);
        count++;
        // setStyle(DefaultsService.getDefaultStyle("SpaceVIL.Frame"));
        setStyle(DefaultsService.getDefaultStyle(Frame.class));
    }

    @Override
    public boolean getHoverVerification(float xpos, float ypos) {
        return false;
    }
}